nway/utils.py

```diff
@@ -91,6 +92,53 @@ def row_col_from_roi(roi):
     return masked
 
 
+def layered_mask_from_rois(rois: List[Any],
+                           shape: Tuple[int, int]) -> np.ndarray:
+    """makes a (nz, *shape) dimensioned array where each layer ("z")
+    has no overlapping ROIs
+
+    Parameters
+    ----------
+    rois: list
+        list of ROIs. The list is the same as "cell_rois" in
+        ExperimentSchema
+    shape: tuple
+        (nrows, ncols) of the destination image
+
+    Returns
+    -------
+    masks: numpy.ndarray
+       dimensions (nz, nrows, ncols)
+
+    """
+    def new_layer():
+        return np.zeros(shape, dtype=np.uint32)
+
+    masks = [new_layer()]
```

Comment:
`mask_layers = [new_layer()]` ?

Not changing is fine too.

Reply:
I think I'll leave it.